draft-welzl-ccwg-constrained-increase.md

```diff
@@ -66,20 +66,28 @@ informative:
 
 --- abstract
 
-This document specifies how transport protocols should increase their congestion window when the sender is constrained. Such a constraint can  be caused by the application stopping to supply data or by flow control.
+This document specifies how transport protocols should increase their congestion window when the sender is constrained.
+Such a constraint can  be caused by the application stopping to supply data or by flow control.
 
 
 --- middle
 
 # Introduction
 
-We call a sender of a congestion controlled transport protocol "constrained" when it does not have data to send (i.e., the application has not provided enough data) even though the congestion control rules would allow it to transmit data. A sender can also be constrained when flow control limits apply (e.g., via the receiver window (rwnd) in case of TCP).
+We call a sender of a congestion controlled transport protocol "constrained" when it does not have data to send (i.e., the application has not provided sufficient data) even though the congestion control rules would allow it to transmit data.
+A sender can also be constrained by the receiver when flow control limits apply (e.g., via the receiver window (rwnd) in case of TCP, of the flow credit in quic).
 RFCs specifying congestion control mechanisms diverge regarding the rules for increasing the congestion window (cwnd) when the sender is constrained.
+This is also known as a rate-limited sender.
```

Comment:
```suggestion
```
I take this out because I'll instead change all the terminology to rate-limited afterwards.